index.css

```diff
@@ -10,6 +10,33 @@ body {
 /*================================================1 - BANNER ========================================*/
 /*================================================================================================*/
 
+/*****************************************   Website header  **************************************/
+.banner-section .web-header {
+  width: 100%;
+  background: black;
+  position: absolute; /* we need it, because otherwise it takes the place below image */
+  top: 0px;
+  backdrop-filter: blur(2px);
+}
+.banner-section .web-header > * {
+  display: inline-block;
+  margin: 14px 0px 14px 220px;
+  color: #ffffff;
+}
+.banner-section .web-header > a {
+  margin: 0px;
+  padding-left: 58px;
+  vertical-align: 8px;
+  text-decoration: none;
+}
+.banner-section .web-header > p {
```

Comment:
anner-section .web-header > and anner-section .web-header > p almost same

Reply:
No! I have in .web-header 4 tags <a> and one <div> with logo, and also i have only one tag <p>, so i need to use styles only for ONE <p>, not for all section.